### DISCUSSION TITLE
Physx 5.1 on non NVidia GPUs?

### DISCUSSION THREAD

**Discussion by cbertin1932** [NVIDIA-Omniverse/PhysX#27] [Q&A] [answered]
I noticed the CUDA manager in the snippets. So my question, since I have 3080 graphics card and can run the snippets, is can Physx 5.1 run on other graphics cards like from AMD?

**Comment by amoravanszky** [comment, marked as answer]
CUDA is only available on NVIDIA GPUs.

**Reply by HaydnVH** [reply]
Can a non-CUDA version of the binaries be made available?
